test_scripts/UnknownEnums/007_multiple_unknown_enums_in_array_of_structs.lua

```diff
@@ -0,0 +1,163 @@
+local putFileParams = {
+  requestParams = {
+    syncFileName = 'icon.png',
+    fileType = "GRAPHIC_PNG",
+    persistentFile = false,
+    systemFile = false
+  },
+  filePath = "files/icon.png"
+}
+
+local requestParams = {
+  mainField1 = "mainField1_text",
+  mainField2 = "mainField2_text",
+  mainField3 = "mainField3_text",
+  mainField4 = "mainField4_text",
+  templateTitle = "templateTitle_text",
+  statusBar = "statusBar_text",
+  mediaClock = "mediaClock_text",
+  mediaTrack = "mediaTrack_text",
+  alignment = "CENTERED",
+  graphic = {
+    imageType = "DYNAMIC",
+    value = "icon.png"
+  },
+  secondaryGraphic = {
+    imageType = "DYNAMIC",
+    value = "icon.png"
+  },
+  metadataTags = {
+    mainField1 = { "mediaTitle" },
+    mainField2 = { "mediaArtist" },
+    mainField3 = { "mediaAlbum" },
+    mainField4 = { "mediaYear" },
+  },
+  softButtons = { 
+    {
+    softButtonID = 1,
+    type = "UNKNOWN_1"
+    },
+    {
+    softButtonID = 2,
+    type = "UNKNOWN_2",
+    text = "Hello"
+    },
+  }
+}
+
+local responseUiParams = {
+  showStrings = {
+    {
+      fieldName = "mainField1",
+      fieldText = requestParams.mainField1,
+      fieldTypes = requestParams.metadataTags.mainField1
+    },
+    {
+      fieldName = "mainField2",
+      fieldText = requestParams.mainField2,
+      fieldTypes = requestParams.metadataTags.mainField2
+    },
+    {
+      fieldName = "mainField3",
+      fieldText = requestParams.mainField3,
+      fieldTypes = requestParams.metadataTags.mainField3
+    },
+    {
+      fieldName = "mainField4",
+      fieldText = requestParams.mainField4,
+      fieldTypes = requestParams.metadataTags.mainField4
+    },
+    {
+      fieldName = "templateTitle",
+      fieldText = requestParams.templateTitle
+    },
+    {
+      fieldName = "mediaClock",
+      fieldText = requestParams.mediaClock
+    },
+    {
+      fieldName = "mediaTrack",
+      fieldText = requestParams.mediaTrack
+    },
+    {
+      fieldName = "statusBar",
+      fieldText = requestParams.statusBar
+    }
+  },
+  alignment = requestParams.alignment,
+  graphic = {
+    imageType = requestParams.graphic.imageType,
+    value = common.getPathToFileInAppStorage(requestParams.graphic.value)
+  },
+  secondaryGraphic = {
+    imageType = requestParams.secondaryGraphic.imageType,
+    value = common.getPathToFileInAppStorage(requestParams.secondaryGraphic.value)
+  }
+}
+
+local allParams = {
+  requestParams = requestParams,
+  responseUiParams = responseUiParams,
+}
+
+--[[ Local Functions ]]
+local function show(pParams)
+  local cid = common.getMobileSession():SendRPC("Show", pParams.requestParams)
+  pParams.responseUiParams.appID = common.getHMIAppId()
+  common.getHMIConnection():ExpectRequest("UI.Show", pParams.responseUiParams)
+  :Do(function(_, data)
+      common.getHMIConnection():SendResponse(data.id, data.method, "SUCCESS", { })
+    end)
+  :ValidIf(function(_, data)
+      return data.params["softButtons"] == nil
```

Comment:
Recently I made changes which make this test fail. Since `softbuttons` is `minsize="0"`, it no longer removes the element and just leaves it empty instead